### ISSUE TITLE
"Range too large for use with labels" error when using `--no-board-labels`

### ISSUE THREAD

**Issue by julianandrews** [julianandrews/sgf-render#47]
For SZ > 99 we throw this error, which makes sense if we are trying to draw board labels (which are limited by space to two characters), but doesn't make sense with `--no-board-labels`.

```
echo '(;GM[1]FF[4]SZ[255];B[bb]W[Bc])' | sgf-render --format text --no-board-labels
Failed to write output: Range too large for use with labels
```

**Comment by julianandrews** [comment]
On the other hand, there is literally no way in the SGF spec to reference a point outside a 52x52 board, so maybe the real issue here is that if either coordinate is larger than 52 we should be returning a more on point error.

**Comment by julianandrews** [comment]
Fixed in 5f38b73. I decided to throw an error for board sizes > 52. I can't come up with a scenario where drawing a larger board than you can place stones on is actually useful, and I think this is probably the best user experience.
